src/test/java/com/jfrog/mavendeptree/integration/Utils.java

```diff
@@ -78,14 +79,30 @@ static List<String> runMavenDepTree(String projectName, String testOutputDir, St
      * @throws IOException           in case of any unexpected I/O error.
      * @throws VerificationException in case of any Maven Verifier error.
      */
-    static List<String> runMavenProjectTree(String projectName, String pluginVersion) throws IOException, VerificationException {
+    static List<String> runMavenProjectTree(String projectName, String pluginVersion, Boolean withOutputFile) throws IOException, VerificationException {
         File testDir = ResourceExtractor.simpleExtractResources(Utils.class, "/integration/" + projectName);
         Verifier verifier = new Verifier(testDir.getAbsolutePath());
+
         if (StringUtils.equalsIgnoreCase(System.getProperty("debugITs"), "true")) {
             verifier.setEnvironmentVariable("MAVEN_OPTS", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005");
         }
-        verifier.executeGoals(Lists.newArrayList("clean", "com.jfrog:maven-dep-tree:" + pluginVersion + ":projects", "-q"));
+
+        String outputFile = verifier.getLogFileName();
+        List<String> goals = Lists.newArrayList("clean", "com.jfrog:maven-dep-tree:" + pluginVersion + ":projects", "-q");
+
+        if (withOutputFile) {
+            goals.add("-DdepsTreeOutputFile=" + testDir.getAbsolutePath() + "/testOutput.out");
```

Comment:
Instead of writing temporary files in the workspace, let's use [Files.createTempFile()](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempFile-java.lang.String-java.lang.String-java.nio.file.attribute.FileAttribute...-) to create a temporary file.